src/Mobs/AggressiveMonster.cpp

```diff
@@ -46,6 +47,36 @@ void cAggressiveMonster::EventSeePlayer(cPlayer * a_Player, cChunk & a_Chunk)
 
 
 
+bool cAggressiveMonster::CanSeeMobType(eMonsterType a_MobType, cAggressiveMonster * a_Monster, int a_SightDistance)
```

Comment:
```suggestion
bool cAggressiveMonster::CanSeeMobType(eMonsterType a_MobType, cAggressiveMonster * a_Monster, unsigned int a_SightDistance)
```

Either add a check that processes negative values or only use unsigned